### ISSUE TITLE
data browser not showing the values of byte[] type

### ISSUE THREAD

**Issue by LeoXJ978** [objectbox/objectbox-java#1033]
When model has a field of byte[], though the values seem correctly persisted, data browser shows blank for the field

**Basic info (please complete the following information):**
 - ObjectBox version (are you using the latest version?):  3.0.1
 - Reproducibility: [e.g. occurred once only | occasionally without visible pattern | always]
 - Device:  Huawei Mate 9
 - OS: Android 9

**To Reproduce**
1. Put a field of byte[]
2. assign some values, then persist the model
3. check the field in data browser

**Expected behavior**
the values should be shown

**Comment by greenrobot-team** [comment]
Confirmed, thanks for reporting!

The values are actually returned from the database, but not in a format that can be displayed. Example:
```
http://localhost:8090/api/query?entity_name=Note&offset=0&limit=50
{ "data":
 {
 "offset": 0,
 "fields": ["id", "text", "date", "bytes", "authorId"],

 "count": 3, "objects": [
  ["1", "This is a note for Bob", "1638789338241", "\u0001\u0002\u0003", "1"],
  ["2", "Write a demo app for ObjectBox", "1638789338242", "\u0001\u0002\u0003", "1"],
  ["3", "Thanks for your note, Alice", "1638789338242", "\u0001\u0002\u0003", "2"]
 ]
}
}
```

**Comment by greenrobot-team** [comment]
This is fixed in the `3.1.0` release. Byte arrays are now shown as Base64 encoded values.
